collectors/crowdstrike/utils.js

```diff
@@ -0,0 +1,126 @@
+const RestServiceClient = require('@alertlogic/al-collector-js').RestServiceClient;
+
+const Incident = 'Incident';
```

Comment:
please use upper case for global const vars.

Reply:
Done